.github/ISSUE_TEMPLATE/Bug_Currency_Dashboard.yml

```diff
@@ -0,0 +1,39 @@
+name: Bug for Currency Dashboard
+title: "[BUG]: "
+description: "Bug reporting for currency dashboard"
+labels: "dashboard bug"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug or feedback reports for Currency Dashboard. Also to request currency for any package or report currency issues for any package.
+        Please fill in the following details to provide bug details for Currency Dashboard.
+  - type: dropdown
+    id: dashboard_version
+    attributes:
+      label: Currency Dashboard Version
+      options:
+        - v0.5
+        - v0.6
+        - v1.0
+      description: |
+        The version of Currency Dashboard on which you see behavior or registering your feedback for dashboard.
```

Comment:
Can we keep this as "Version of the currency dashboard that you are seeing the bug on or submitting feedback for"

Reply:
Agreed on Priya's comment - can we also have the current version set as the default answer?  I'm trying to think of a case where someone could report something with an older version - that seems difficult?  But we definitely want to save the version in which they reported the issue, e.g. if we don't fix a bug for 6 months, it may be important to know it was filed against an older version of the portal.

Reply:
sure changing `Version of the currency dashboard that you are seeing the bug on or submitting feedback for`

Reply:
On question about default, whatever first value we keep in options , comes as default in form. So whenever we release new version we can update this template have latest version on top.
![image](https://github.com/ppc64le/build-scripts/assets/82815888/8dac0ab8-93be-40af-b404-ef4d42a894f5)